tf_parent_frame_id: "testbed_origin"
tf_child_frame_id: "pouring_point"

plate_tip_position:
  x: 0.305
  y: 0.305

plate_orientation:
    x: 0.0
    y: 0.0
    z: 0.0
    w: 1.0

pouring_offset_position:
  x: 0
  y: -0.05

plate_angles: [15., 30., 45., 60., 75.] # deg
plate_tip_heights: [0.153, 0,215, 0.308, 0.376, 0.394] # m

# During the tests, adjust these for different experiments
desired_pouring_height: 0.05 # m
plate_selection_index: 4  # which config is chosen above [0,1,2,3,4]
